docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,673 @@
+
+The most requested feature for the dashboard is to manage the lifecycles of your orchestrated named resources. Specifically, the ability to stop, start, and restart resources. This feature is now available in .NET Aspire 9.
+
+<!-- markdownlint-disable MD033 -->
+<video controls src="videos/start-stop-restart.mp4" title="Title"></video>
+<!-- markdownlint-enable MD033 -->
+
+This feature works for projects, containers and executables. It enables restarting indidivual resources without having to restart the entire app host. For projects, if the debugger is attached, on restart, the debugger will be re-attached.
+
+- Responsive design
+- Mobile friendly
+
+### Sensitive properties, volumes and health checks in resource details
+
+Resource details has seen a lot of improvements:
+
+- Properties can be marked as sensitive, automatically masking them in the dashboard UI. This is a security feature to avoid accidently disclosing keys or passwords when screen sharing the dashboard with other people. For example, container arguments could pass sensitive information and so are masked by default.
+
+- Configured container volumes are listed in resource details.
+
+- Detailed information about resource health checks, a new feature in Aspire 9, are listed in resource details. This information is useful to understand why a resource might be marked as unhealthy. Find out more about health checks [here](#resource-health-checks).
+
+### Coloreful console log
+
+[ANSI escape codes](https://wikipedia.org/wiki/ANSI_escape_code) are used to add text formatting to text. For example, foreground and background colors, bold, underline, italics, etc. The dashboard console logs page previously supported rendering one ANSI escape code at a time and failed when they were combined together. The console logs page could show red text, but it couldn't show text that is red and bold at the same time.
+
+A community contribution from [@mangeg](https://github.com/mangeg) improved support for ANSI escape codes and removed this limitation. Behold:
+
+![Colorful console logs](images/console-logs-ansi-text-format.png)
+
+Another improvement to console logs is hiding unsupported escape codes. Codes that aren't related to displaying text, such as positioning the cursor or communicating with the operating system don't make sense in this UI, and are hidden.
+
+## Telemetry
+
+### Improve telemetry filtering
+
+Traces can now be filtered with attribute values. For example, if you only want to view traces for one endpoint in your app, the `http.route` attribute on HTTP requests can be filtered to a specified value.
+
+Telemetry filtering also supports auto-complete of existing values. The **Add filter** dialog provides a combo box for selecting from values that dashboard has received. This feature makes it much easier to filter to real data and helps avoid typos by entered a value yourself.
+
+<video controls src="videos/trace-filter.mp4" title="Video of filtering traces"></video>
```

Comment:
I added this video but I don't see it in the preview. Usually I use gifs rather than video for animations so I'm not sure about the process here.

Why isn't it visible?

cc @IEvangelist

Reply:
We can't reference videos in this way. They have to be uploaded to the Learn [video server](https://review.learn.microsoft.com/en-us/help/platform/video-service-upload?branch=main&tabs=add%2Cbrowser%2Cframe).